autotx/cli.py

```diff
@@ -77,7 +80,8 @@ def run(prompt: str | None, non_interactive: bool, verbose: bool, logs: str | No
 def serve(verbose: bool, logs: str | None, max_rounds: int | None, cache: bool, port: int | None, dev: bool) -> None:
     print_autotx_info()
 
-    start_server(verbose, logs, max_rounds, cache, port, dev)
+    setup_server(verbose, logs, max_rounds, cache, dev)
+    uvicorn.run("autotx.server:app", host="localhost", port=8000, workers=1)
```

Comment:
After going through the codebase more, I can see how it might be a significant lift to make AutoTx run within a serverless configuration. If it can't be done in ~1 day's time, then I think we ship this first version of the discord bot as is with a persistent service approach.

But if there are immediate issues with the agent's functionality related to this (ex: queue is overwhelmed too quickly) then it's a must have. Otherwise, we can work on serverless scaling immediately after.